albumentations/core/composition.py

```diff
@@ -183,6 +185,9 @@ def _disable_check_args_for_transforms(transforms: TransformsSeqType) -> None:
     def _disable_check_args(self) -> None:
         self.is_check_args = False
 
+    def disable_shapes_check(self) -> None:
+        self.is_check_shapes = False
```

Comment:
I think we must control this behavior using `is_check_shapes` as common argument inside `__init__`